.golangci.yml

```diff
@@ -0,0 +1,36 @@
+linters-settings:
+  govet:
+    check-shadowing: true
+
+linters:
+  disable-all: true
+  enable:
+    - unused
+    - gosimple
+    - structcheck
+    - varcheck
+    - ineffassign
+    - deadcode
+    - unconvert
+    # TODO: enable these later
+    # - govet
+    # - gofmt
+    # - errcheck
+    # - staticcheck
+    # - unparam
+
```

Comment:
Would you please set `modules-download-mode` to `vendor`. This way we are sure to use the same deps as the build scripts.

Reply:
Gah, missed that. Thanks !